gmsh_interop/runner.py

```diff
@@ -386,7 +396,7 @@ def __enter__(self) -> GmshRunner:
                         shutil.copytree(working_dir, self.save_tmp_files_in)
```

Comment:
Does this `copytree` even work? It seems like it would just do the same thing as above, and fail in the same way?

Reply:
Looks like it calls `rmtree` above that after waiting for input from the user.